config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  get 'private/test'
+
+  # root "articles#index"
```

Comment:
- [ ] Kindly, remove this old piece of commented code, in order to preserve the repo's cleanliness. 👍